common/src/test/kotlin/io/homeassistant/companion/android/common/ConsoleLogTree.kt

```diff
@@ -0,0 +1,19 @@
+package io.homeassistant.companion.android.common
+
+import timber.log.Timber
+
+object ConsoleLogTree : Timber.DebugTree() {
+    var verbose = false
+
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+        if (t != null) {
+            System.err.println("[$tag] - $message - ${t.stackTraceToString()}")
+        } else {
+            if (verbose) {
+                System.err.println("[$tag] - $message")
+            } else {
+                // no-op
+            }
+        }
```

Comment:
Without this there are no logs available in the AS console. First because Timber needs a tree where to send logs.
Then we need to use stderr because Junit is forwarding stdout to build the reports.... I can add a comments if you find it useful.